### DISCUSSION TITLE
Share screen when displaying media

### DISCUSSION THREAD

**Discussion by UNiXMIT** [sircharlo/meeting-media-manager#1962] [Ideas] [answered]
I love the new Zoom integration. I love this app but thought adding OBS into the mix just added more complexity. I'm hoping this Zoom integration will remove the need for OBS. Currently, is there an option to make Zoom share a particular screen when you click the button to display media in M3? That would be great if it could share a set screen with audio sharing enabled when you play a song in M3 and also just share a set screen when displaying images from M3. Is that possible or on the roadmap to implement? Thanks so much for all your hard work.

**Comment by mtdvlpr** [comment, marked as answer]
Hi @UNiXMIT, we're using the [Zoom Web SDK](https://marketplacefront.zoom.us/sdk/meeting/web/components/modules/EmbeddedClient.html) which sadly doesn't have a function for screen sharing.

**Reply by UNiXMIT** [reply]
> Hi @UNiXMIT, we're using the [Zoom Web SDK](https://marketplacefront.zoom.us/sdk/meeting/web/components/modules/EmbeddedClient.html) which sadly doesn't have a function for screen sharing.

Hmm yeah true. Thanks anyway.
